lms/templates/dashboard.html

```diff
@@ -49,31 +47,8 @@
       });
     });
   </script>
-  <%static:webpack entry="UnenrollmentFactory">
-    UnenrollmentFactory({
-      urls: {
-        dashboard: "${reverse('dashboard') | n, js_escaped_string}",
-        signInUser: "${reverse('signin_user') | n, js_escaped_string}",
-        changeEmailSettings: "${reverse('change_email_settings') | n, js_escaped_string}",
-        browseCourses: "${marketing_link('COURSES') | n, js_escaped_string}"
-      },
-      isEdx: false
-    });
-  </%static:webpack>
-  <%static:webpack entry="EntitlementUnenrollmentFactory">
-    ## Wait until the document is fully loaded before initializing the EntitlementUnenrollmentView
-    ## to ensure events are setup correctly.
-    $(document).ready(function() {
-      EntitlementUnenrollmentFactory({
-        dashboardPath: "${reverse('dashboard') | n, js_escaped_string}",
-        signInPath: "${reverse('signin_user') | n, js_escaped_string}",
-        browseCourses: "${marketing_link('COURSES') | n, js_escaped_string}",
-        isEdx: false
-      });
-    });
-  </%static:webpack>
   % if settings.FEATURES.get('ENABLE_DASHBOARD_SEARCH'):
-    <%static:require_module module_name="course_search/js/dashboard_search_factory" class_name="DashboardSearchFactory">
+    <%static:require_module module_name="js/search/dashboard/dashboard_search_factory" class_name="DashboardSearchFactory">
```

Comment:
Let's please make sure this module exists at the mentioned path, becuase we've pasted the whole file from ficus, so we should verify all the paths